### ISSUE TITLE
Unknown props in React v15.2

### ISSUE THREAD

**Issue by Khrall** [okonet/react-dropzone#193]
Just noticed this warning when using version 15.2 of React:

```
Warning: 
Unknown props `disablePreview`, `disableClick` on <div> tag. 
Remove these props from the element. 
For details, see https://fb.me/react-unknown-prop
    in div (created by Dropzone)
    in Dropzone (created by xxx)
```

The [link provided](https://fb.me/react-unknown-prop) seems to indicate that the problem is at line 219 of index.js.

```
216 <div
    ...
219 {...props /* expand user provided props first so event handlers are never overridden */}
    ...
226 >
```

Anyway, I don't know much about this stuff, might be I screwed up something else. Just wanted to let you know in case it's going to show up for other people (:

**Comment by OlivierCo** [comment]
<img width="1398" alt="screen shot 2016-07-04 at 12 32 58" src="https://cloud.githubusercontent.com/assets/1804322/16559234/78594f68-41e3-11e6-80c9-ffa1b1af64c4.png">

**Comment by okonet** [comment]
Fixed in #194

**Comment by okonet** [comment]
Please update to [v3.5.2](https://github.com/okonet/react-dropzone/releases/tag/3.5.2)

**Comment by mikecousins** [comment]
Beautiful, thanks for the quick work.

**Comment by okonet** [comment]
A follow up fix in #197
